corehq/apps/es/management/commands/elastic_sync_multiplexed.py

```diff
@@ -32,31 +36,65 @@ def start_reindex(self, cname):
             raise IndexNotMultiplexedException("""Index not multiplexed!
             Sync can only be run on multiplexed indices""")
 
-        self.source, self.dest = self._get_source_destination_indexes(adapter)
-
-        task_id = es_manager.reindex(self.source, self.dest)
-
+        source_index, destination_index = self._get_source_destination_indexes(adapter)
+
+        logger.info(f"Preparing index {destination_index} for reindex")
+        self._prepare_index_for_reindex(destination_index)
+
+        logger.info("Starting ReIndex process")
+        task_info = es_manager.reindex(source_index, destination_index)
+        logger.info(f"Copying docs from index {source_index} to index {destination_index}")
+        task_id = task_info.split(':')[1]
+        print("\n\n\n")
```

Comment:
Mixing logging and print statements is a little weird. Is this intended to conserve space in the log file while making the console output more readable, or is there another reason?

Reply:
It was to make console output more readable. This is how the output looks

![Screenshot 2023-06-08 at 10 53 22 PM](https://github.com/dimagi/commcare-hq/assets/7694243/dc3a61d8-c0e9-4827-9498-0c099abbd790)